MUXSDKImaListener/Classes/include/MuxImaListener.h

```diff
@@ -25,6 +25,7 @@ typedef NS_OPTIONS(NSUInteger, MuxImaListenerOptions) {
     @private BOOL _isPictureInPicture;
     @private BOOL _usesServerSideAdInsertion;
     @private BOOL _adRequestReported;
+    @private BOOL _sendAdplayOnStarted;
```

Comment:
These private members should be defined as properties in a class extension.

We've added the others awhile ago so it makes sense for them to be private instance variables, but not long after after Obj-C received support for ARC, classes could define internal members as properties using the class extension mechanism. https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW3

I can help fix the other already existing private members in a separate PR.

Reply:
Thanks for offering to fix the rest! I update this one